template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Data/ApplicationDbContextFactory.cs

```diff
@@ -9,7 +9,7 @@ namespace Company.WebApplication1.Data
 {
     public class ApplicationDbContextFactory : IDbContextFactory<ApplicationDbContext>
     {
-        public ApplicationDbContext Create(DbContextFactoryOptions options) =>
-            Program.BuildWebHost(Array.Empty<string>()).Services.GetRequiredService<ApplicationDbContext>();
+        public ApplicationDbContext Create(string[] args) =>
+            Program.BuildWebHost(args).Services.GetRequiredService<ApplicationDbContext>();
```

Comment:
May change our signature to `IEnumerable<string>`. Should I call `.ToArray()` here or change the signature of `BuildWebHost` too?